package.json

```diff
@@ -108,7 +109,9 @@
     "nyc": "^15.0.1",
     "prettier": "^2.0.4",
     "tmp": "^0.1.0",
+    "type-fest": "^0.13.1",
```

Comment:
I don't know why we need this, probably not needed with gts.

Reply:
`type-fest` is currently only used for some advanced types. `ReadonlyDeep` comes to mind.